service/models/converters.py

```diff
@@ -11,6 +11,13 @@ def convert_to_static_interventions(interventions):
     return static_interventions
 
 
+def convert_optimize_to_static_interventions(interventions):
```

Comment:
I cannot overload based off of type provided in python correct?

Reply:
correct